### ISSUE TITLE
Popover component

### ISSUE THREAD

**Issue by hillin** [Megabit/Blazorise#3707]
Since it's quite useful and have been there in bootstrap forever, I think it's good to have this component in Blazorise too.

**Comment by hillin** [comment]
@stsrki good point! Thanks.

**Comment by hillin** [comment]
@stsrki Apparently the Tooltip component does not support customizing its content (only a text content is supported). Sounds like a FR should be proposed to extend this feature?

**Comment by stsrki** [comment]
Tooltip allows to have an html text as a content so you should be able to do it.

**Comment by hillin** [comment]
But not a blazor component I suppose?

**Comment by stsrki** [comment]
No, the component is not possible to have as a tooltip content. I don't think it will ever be possible.
